sdk/textanalytics/azure-ai-textanalytics/samples/async_samples/sample_recognize_entities_async.py

```diff
@@ -21,17 +21,17 @@
 
 OUTPUT:
     Document text: Microsoft was founded by Bill Gates and Paul Allen.
-    Entity:          Microsoft      Type:    Organization   Confidence Score:        1.0
-    Entity:          Bill Gates     Type:    Person         Confidence Score:        1.0
-    Entity:          Paul Allen     Type:    Person         Confidence Score:        1.0
+    Entity:          Microsoft      Category:    Organization   Confidence Score:        1.0
```

Comment:
I'm wondering, if for whatever reason the service changes the model, if the confidence score number will change. 
So if a customer runs the above sample code and compares the output, will find different results and get confused.

Reply:
Yeah this has been on my mind for awhile. I like having the output because the user can see what to expect without running, but in the case of this service (where results will change over time), I think it might not make sense / cause confusion like you said. I'm going to remove this section in the samples, thanks for bringing it up.